configuration-example.js

```diff
@@ -1,3 +1,27 @@
 const EluvioConfiguration = {
-  "config-url": "https://main.net955304.contentfabric.io/config"
-};
+  // Fabric Network
+  "config-url": "https://test.net955203.contentfabric.io/config",
```

Comment:
after looking at the code, fabric browser doesn't actually need config-url  so we can remove that from the config and references to it from the readme.